src/scheduler/VertexScheduler/VertexJobRuns.tsx

```diff
@@ -0,0 +1,340 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useEffect, useState } from 'react';
+import { useTable, useGlobalFilter } from 'react-table';
+import { CircularProgress } from '@mui/material';
+import { Dayjs } from 'dayjs';
+
+import TableData from '../../utils/tableData';
+import { ICellProps, handleDebounce } from '../../utils/utils';
+import { SchedulerService } from '../schedulerServices';
+import { VertexServices } from './VertexServices';
+import { IconDownload } from '../../utils/icons';
+
+interface IDagRunList {
```

Comment:
Move this to common file and use it where ever required. This interface is repeated.